src/components/Chip/KeywordChip.jsx

```diff
@@ -0,0 +1,13 @@
+import PropTypes from "prop-types";
+
+const KeywordChip = ({ keywordName }) => {
+  return (
+    <span className="flex-center text-14 px-3 py-2 bg-orange-200 rounded-[3px]">{keywordName}</span>
+  );
+};
+
+export default KeywordChip;
+
+KeywordChip.propTypes = {
+  keywordName: PropTypes.string,
```

Comment:
```suggestion
  keywordName: PropTypes.string.isRequired,
```
해당 prop을 필수로 명시하면 어떨까요? 현재 쓰이고 있는 곳이 `UserGroupCard` 컴포넌트가 유일한데, `keywordName`을 prop으로 내려주고 있더라구요. 그리고 저희가 추후 활용할 컴포넌트에서도 항상 필요할 것 같습니다! (e.g. 추가/키워드 편집 모달에서 '현재 등록된 키워드 Chip')

```js
// 현재 UserGroupCard에서 Chip 활용되는 부분
{keywordList.map((keywordName) => {
   <KeywordChip key={keywordName} keywordName={keywordName} />;
})}
```

Reply:
네 반영했습니다 :)